docs/prompt-library/prompt-library.md

```diff
@@ -2,15 +2,18 @@ This page contains a collection of prompts that you can use as is or design your
 
 <h3>Using the Prompts</h3>
 
-Users can view and utilize the full prompts by visiting the GitHub repository or cloning it locally. The prompts are available in the `prompts.ts` file within the repository.
+Users can view and utilize the full prompts by visiting the GitHub repository or cloning it locally. The prompts are available in the `prompt-library` folder within the repository.
 
 To access the prompts:
 
-1. Visit the GitHub repository: [Link to GitHub repository](https://github.com/awslabs/agents-for-amazon-bedrock-blueprints){:target="_blank"}
+1. Visit the GitHub repository: <a href="https://github.com/aws-samples/amazon-bedrock-samples/tree/main/agents-for-bedrock/agent-blueprint-templates/lib/prompt_library" target="_blank">Link to GitHub repository</a>
```

Comment:
Could you use markdown syntax for consistency?

Reply:
I made the switch to using <a> tags because the link on line 32 for whatever reason was not getting picked up properly by the markdown syntax, and it was displaying the literal text. I even tried copying the original markdown link that was already in the file and it was not getting picked up. So I switched them to use <a> tags for consistency

Reply:
Actually, I think it is just an issue with whatever is converting the Markdown. Let me update both the links to use MD syntax